doc/api/next_api_changes/autoscale_for_arrow.rst

```diff
@@ -0,0 +1,3 @@
+Autoscale for arrow
```

Comment:
this needs to go to https://github.com/matplotlib/matplotlib/blob/master/doc/api/next_api_changes/behaviour.rst now.